### PR TITLE
Mc redis clear

### PR DESCRIPTION
**Pull request by FBRTMaka**
@DanielJMaher , @oceanzus , @birdage , @Bobfrat 

New page that will give a system admin the privileges to delete individual cache items.  Corresponding PR is open for the UI side.

### CODE REVIEW

File: ooiservices/tests/test_sys.py [hunk 1]
```diff
@@ -41,8 +42,30 @@ def get_api_headers(self, username, password):
...
-        self.assertIn('routes', data)
\ No newline at end of file
+        self.assertIn('routes', data)
+
+    '''
+    The redis cache clear route is defined by:
+    '''
+    def test_clear_redis(self):
+        # setup a test cache list
+        cache.set('test_list', [{'something': 'something'}, {'dark': 'side'}])
```
Comment (birdage) [inline ooiservices/tests/test_sys.py]: :+1:

**Comment by DanielJMaher** [comment]
@FBRTMaka Is this good to go or is it still under construction?

**Comment by FBRTMaka** [comment]
@DanielJMaher good. to. go. bro.

**Comment by DanielJMaher** [comment]
@oceanzus This looks good to me, but lets have Jim make the final call here.

**Comment by oceanzus** [comment]
Merge it.

**Pull request merged**